addon/components/bsy-radio-buttons/button.js

```diff
@@ -11,7 +11,7 @@ const BsyRadioButtonsButtonComponent = Component.extend({
 
   activeClassName: computed('value', function() {
     return isEqual(this.get('value'), this.get('option'));
```

Comment:
This is wrong in several ways. I believe we don't need to `.get()` and a dependent key is probably missing.